utils/plugin/plugin_test.go

```diff
@@ -0,0 +1,52 @@
+package plugin
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/argoproj/argo-rollouts/utils/defaults"
+
+	"github.com/argoproj/argo-rollouts/utils/config"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestGetPluginLocation(t *testing.T) {
```

Comment:
We use mockery to auto generate mocks from interfaces in Argo CD. This is something that you can have taken advantage as well to implement more test cases. Not a blocker, mainly a suggestion to improve the coverage.